content/docs/getting-started/migrate-jasmin-account-from-ceda.md

```diff
@@ -7,64 +7,13 @@ tags:
 weight: 190
```

Comment:
I wasn't sure if you would prefer to scrap this whole page.

I left it so as to not create dead links from elsewhere... But a good part of me would prefer to just delete it completely! Let me know if you prefer that option.